packages/paste-core/components/breadcrumb/src/index.tsx

```diff
@@ -57,44 +72,50 @@ BreadcrumbItem.displayName = 'BreadcrumbItem';
 if (process.env.NODE_ENV === 'development') {
   BreadcrumbItem.propTypes = {
     children: PropTypes.node.isRequired,
+    element: PropTypes.string,
     href: PropTypes.string,
     last: PropTypes.bool,
   };
 }
 
 export interface BreadcrumbProps extends React.HTMLAttributes<'nav'> {
   children: NonNullable<React.ReactNode>;
+  element?: BoxElementProps['element'];
 }
 
-const Breadcrumb = React.forwardRef<HTMLDivElement, BreadcrumbProps>(({children, ...props}, ref) => {
-  const [childrenCount, validChildren] = React.useMemo(
-    () => [
-      React.Children.count(children),
-      React.Children.toArray(children).filter((child) => React.isValidElement(child) || typeof child === 'string'),
-    ],
-    [children]
-  );
-  const keySeed = useUIDSeed();
+const Breadcrumb = React.forwardRef<HTMLDivElement, BreadcrumbProps>(
+  ({children, element = DEFAULT_ELEMENT_NAME, ...props}, ref) => {
+    const [childrenCount, validChildren] = React.useMemo(
+      () => [
+        React.Children.count(children),
+        React.Children.toArray(children).filter((child) => React.isValidElement(child) || typeof child === 'string'),
+      ],
+      [children]
+    );
+    const keySeed = useUIDSeed();
 
-  return (
-    <Box {...safelySpreadBoxProps(props)} aria-label="breadcrumb" as="nav" ref={ref}>
-      <Box alignItems="center" as="ol" display="inline-flex" listStyleType="none" margin="space0" padding="space0">
-        {validChildren.map((child, index) =>
-          React.cloneElement(child as React.ReactElement<any>, {
-            last: childrenCount === index + 1,
-            key: keySeed(`breadcrumb-${index}`),
-          })
-        )}
+    return (
+      <Box {...safelySpreadBoxProps(props)} aria-label="breadcrumb" as="nav" element={element} ref={ref}>
```

Comment:
Wouldn't this Box end up having the same `data-paste-element` value as the BreadCrumbSeparator? `DEFAULT_ELEMENT_NAME`?

Reply:
Separator does a concat

Reply:
Oh I see that now